pypeit/core/wavecal/wvutils.py

```diff
@@ -31,6 +31,38 @@ def parse_param(par, key, slit):
 
     return param
 
+
+def get_delta_wave(wave, gpm):
+    """
+    Given an input wavelength vector and an input good pixel mask, this code computes the delta_wave defined
+    to be wave_old[i]-wave_old[i-1]. The missing point at the end of the array is just appended to have an
+    equal size array.
+
+    Args:
+        wave (float ndarray): shape = (nspec,)
```

Comment:
ndarray link and is float meant to specify the data type or
that a float can alternatively be input?  Clarify here and
elsewhere.

Reply:
Fixed.